tests/test_repo/broken_module/security/ir.model.access.csv

```diff
@@ -1,2 +1,5 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_test_model","access_test_model","model_test_model","base.group_user",1,1,1,1
+"access_test_model2","access_test_model2","model_test_model2","base.group_user",1,1,1,1
+"access_test_model2","access_test_model3","model_test_model3","base.group_user",1,1,1,1
+"access_test_model4","access_test_model4","model_test_model4","base.group_user",1,1,1,1

```

Comment:
That will raise an error, but for the wrong reason (invalid model), thus not performing the check on the error we want. You're not testing the duplicate ID.